MatrixSDK/MXRestClient.m

```diff
@@ -2357,6 +2357,62 @@ - (MXHTTPOperation*)messagesForRoom:(NSString*)roomId
                                  }];
 }
 
+- (MXHTTPOperation*)threadsForRoom:(NSString*)roomId
```

Comment:
Thinking from the perspective of someone else using the library, if this is called with a `nil` filter then is it going to return the same messages as `messagesForRoom`? So should this be adding the `relationTypes` to the filter (creating if needed) for you? That's if I have interpreted this correctly.

Reply:
Method removed.